bindgen/raze/crates.bzl

```diff
@@ -0,0 +1,415 @@
+        build_file = Label("//bindgen/raze/remote:clang-sys-0.23.0.BUILD")
+    )
+
+    _new_http_archive(
+        name = "raze__clap__2_32_0",
+        url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/clap/clap-2.32.0.crate",
+        type = "tar.gz",
+        sha256 = "b957d88f4b6a63b9d70d5f454ac8011819c6efa7727858f458ab71c756ce2d3e",
+        strip_prefix = "clap-2.32.0",
+        build_file = Label("//bindgen/raze/remote:clap-2.32.0.BUILD")
+    )
+
+    _new_http_archive(
+        name = "raze__env_logger__0_5_13",
+        url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/env_logger/env_logger-0.5.13.crate",
+        type = "tar.gz",
+        sha256 = "15b0a4d2e39f8420210be8b27eeda28029729e2fd4291019455016c348240c38",
+        strip_prefix = "env_logger-0.5.13",
+        build_file = Label("//bindgen/raze/remote:env_logger-0.5.13.BUILD")
+    )
+
+    _new_http_archive(
+        name = "raze__glob__0_2_11",
+        url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/glob/glob-0.2.11.crate",
+        type = "tar.gz",
+        sha256 = "8be18de09a56b60ed0edf84bc9df007e30040691af7acd1c41874faac5895bfb",
+        strip_prefix = "glob-0.2.11",
+        build_file = Label("//bindgen/raze/remote:glob-0.2.11.BUILD")
+    )
+
+    _new_http_archive(
+        name = "raze__humantime__1_2_0",
+        url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/humantime/humantime-1.2.0.crate",
+        type = "tar.gz",
+        sha256 = "3ca7e5f2e110db35f93b837c81797f3714500b81d517bf20c431b16d3ca4f114",
+        strip_prefix = "humantime-1.2.0",
+        build_file = Label("//bindgen/raze/remote:humantime-1.2.0.BUILD")
+    )
+
+    _new_http_archive(
+        name = "raze__lazy_static__1_2_0",
+        url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/lazy_static/lazy_static-1.2.0.crate",
+        type = "tar.gz",
+        sha256 = "a374c89b9db55895453a74c1e38861d9deec0b01b405a82516e9d5de4820dea1",
+        strip_prefix = "lazy_static-1.2.0",
+        build_file = Label("//bindgen/raze/remote:lazy_static-1.2.0.BUILD")
+    )
+
+    _new_http_archive(
+        name = "raze__libc__0_2_48",
+        url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/libc/libc-0.2.48.crate",
+        type = "tar.gz",
+        sha256 = "e962c7641008ac010fa60a7dfdc1712449f29c44ef2d4702394aea943ee75047",
+        strip_prefix = "libc-0.2.48",
+        build_file = Label("//bindgen/raze/remote:libc-0.2.48.BUILD")
+    )
+
+    _new_http_archive(
+        name = "raze__libloading__0_5_0",
+        url = "https://crates-io.s3-us-west-1.amazonaws.com/crates/libloading/libloading-0.5.0.crate",
+        type = "tar.gz",
+        sha256 = "9c3ad660d7cb8c5822cd83d10897b0f1f1526792737a179e73896152f85b88c2",
+        strip_prefix = "libloading-0.5.0",
+        # TODO: This is a manual patch, need https://github.com/google/cargo-raze/issues/58
+        build_file = Label("//bindgen/raze:libloading.BUILD")
```

Comment:
Maybe this patch should live outside of this file, just so that it doesn't seem hidden?

That can be accomplished by declaring "raze__libloading__0_5_0" ahead of time, thanks to usage of "_new_http_archive". Best place is probably in "rust_bindgen_repositories" just before calling this.

Reply:
Sure I can do that. 

Small tangent.. do you have an idea of how complicated https://github.com/google/cargo-raze/issues/58 is? Would like to handle this in a way that works more closely to the `raze = "Vendor"` setup.